src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .filter((style) => style.includes(':'))
+    .reduce((stylesObject, style) => {
+      const [key, value] = style.split(':').map((item) => item.trim());
```

Comment:
Splitting the style string by colon and trimming each part is a good approach to extract the key and value.